src/js/providers/video-listener-mixin.js

```diff
@@ -163,18 +164,25 @@ const VideoListenerMixin = {
         }
     },
     error() {
-        const code = (this.video.error && this.video.error.code) || -1;
-        const message = ({
-            1: 'Unknown operation aborted',
-            2: 'Unknown network error',
-            3: 'Unknown decode error',
-            4: 'File could not be played'
-        }[code] || 'Unknown');
-        this.trigger(MEDIA_ERROR, {
-            code: code,
-            message: 'Error loading media: ' + message
-        });
+        const errorCode = (this.video.error && this.video.error.code) || -1;
+        const [message, code] = ({
+            1: ['Unknown operation aborted', 1],
+            2: ['Unknown network error', 0],
```

Comment:
A `video.error.code` of `2` maps to `221000` (Unknown network error).

`video.error.code` 1, 3, 4 and other are correct `224xxx` (media errors).